UIKitTemplate/UIKitTemplate/AchorViewController.swift

```diff
@@ -0,0 +1,95 @@
+// AchorViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран с якорями
+final class AchorViewController: UIViewController {
+    // MARK: - Visual Components
+
+    private var redSquareView: UIView = {
+        let red = UIView()
+        red.backgroundColor = .red
+        return red
+    }()
+
+    private var yellowSquareView: UIView = {
+        let red = UIView()
+        red.backgroundColor = .yellow
+        return red
+    }()
+
+    private var greenSquareView: UIView = {
+        let red = UIView()
+        red.backgroundColor = .green
+        return red
+    }()
+
+    private var blackSquareView: UIView = {
+        let red = UIView()
+        red.backgroundColor = .black
+        return red
+    }()
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureUI()
+    }
+
+    // MARK: - Private Methods
+
+    private func configureUI() {
+        view.addSubview(blackSquareView)
+        view.addSubview(yellowSquareView)
+        view.addSubview(redSquareView)
+        view.addSubview(greenSquareView)
+        makeAnchor()
+    }
+
+    private func makeAnchor() {
+        makeYellowSqureAnchor()
+        makeRedSqureAnchor()
+        makeGreenSqureAnchor()
+        makeBlackSqureAnchor()
+    }
+}
+
+// MARK: - Layoyt
+
+extension AchorViewController {
+    private func makeYellowSqureAnchor() {
+        yellowSquareView.translatesAutoresizingMaskIntoConstraints = false
+        yellowSquareView.centerYAnchor.constraint(equalTo: view.centerYAnchor).isActive = true
+        yellowSquareView.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        yellowSquareView.heightAnchor.constraint(equalTo: view.heightAnchor, multiplier: 1 / 4).isActive = true
```

Comment:
```suggestion
        yellowSquareView.heightAnchor.constraint(equalTo: view.heightAnchor, multiplier: 1 / 4).isActive = true
```
такие числа надо в константы выносить вроде, имей ввиду) например 4 заставляли вроде